src/frontend/js/pages/DashboardCertificates/index.tsx

```diff
@@ -22,25 +24,37 @@ interface DashboardCertificatesProps {
 }
 
 export const DashboardCertificates = ({ certificateType }: DashboardCertificatesProps) => {
+  const {
+    items: enrollmentCertificates,
+    states: { isFetched },
+  } = useCertificates({
+    type: CertificateType.ENROLLMENT,
+    page: 1,
+    page_size: PER_PAGE.certificateList,
+  });
+
   return (
     <div className="dashboard-certificates">
-      <Tabs
-        initialActiveTabName={
-          certificateType === CertificateType.ORDER
-            ? 'order-certificate-tab'
-            : 'enrollment-certificate-tab'
-        }
-      >
-        <Tabs.Tab name="order-certificate-tab" href={LearnerDashboardPaths.ORDER_CERTIFICATES}>
-          <FormattedMessage {...messages.orderCertificateTabLabel} />
-        </Tabs.Tab>
-        <Tabs.Tab
-          name="enrollment-certificate-tab"
-          href={LearnerDashboardPaths.ENROLLMENT_CERTIFICATES}
+      {isFetched && enrollmentCertificates.length > 0 && (
+        <Tabs
+          initialActiveTabName={
+            certificateType === CertificateType.ORDER
+              ? 'order-certificate-tab'
+              : 'enrollment-certificate-tab'
+          }
```

Comment:
I know this is related to the previous PR, but I wonder if we could not simply use `certificateType` as tab name.

```suggestion
         initialActiveTabName={certificateType}
```